Sources/Tracking/Background Queue/Type/QueueTaskType.swift

```diff
@@ -0,0 +1,3 @@
+import Foundation
+
+public typealias QueueTaskType = String
```

Comment:
kind of like the idea of making this an enum instead, since we should be able to clearly indicate which types the queue supports & don't need this to be super generic

Reply:
I agree. A `String` is used because strings can be persisted to the file system but enums directly can't. 

So, I was planning on doing both - enum and string:

```swift
enum QueueTaskTypes: String {
   case identifyProfile
   ...
}
```